tools/test/windows/tw.cc

```diff
@@ -492,6 +550,26 @@ bool TouchFile(const std::wstring& path) {
   return true;
 }
 
+bool ReadCompleteFile(HANDLE handle, uint8_t* dest, DWORD max_read_bytes) {
+  if (max_read_bytes == 0) {
+    return true;
+  }
+
+  const DWORD max_read = std::min(
+      max_read_bytes, /* 100 MB */ static_cast<DWORD>(100000000));
```

Comment:
What undeclared outputs are possible, is any of them going to exceed 100MB?

Reply:
Tests are free to write anything into `$TEST_UNDECLARED_OUTPUTS_DIR`.
In theory they could dump large files there.

Reply:
Actually, this restriction makes no sense. There's no reason to limit single ReadFile calls to 100 MB. Let me remove this code.